lib/cocoapods/bazel/xcconfig_resolver.rb

```diff
@@ -8,7 +8,11 @@ module XCConfigResolver
       def resolved_build_setting_value(setting, settings:)
         return unless (value = settings[setting])
 
-        sub_prefix = ->(s) { s.sub(%r{\A:/}, '') }
+        sub_prefix = ->(s) {
+          str_value = s.to_s
+          str_value.sub(%r{\A:/}, '')
+          return str_value
+        }
```

Comment:
Why not keep this in one line `sub_prefix = ->(s) { s.to_s.sub(%r{\A:/}, '') }`? I would split this into more lines only if you had the intent of further manipulating the intermediary values

Reply:
fixed